### PR TITLE
[need-fix][renec] Fix/sorted token before getting pool

### PR DESCRIPTION
**Pull request by kientt6034**
## Vivify ticket

  *`[BDFR-393](https://app.vivifyscrum.com/boards/114704/sprint-backlog/373800/BDFR-393)`*

## PR Dependencies 
https://github.com/renec-chain/nemo-swap/pull/11
https://github.com/renec-chain/nemo-swap/pull/12

## Note
- Get correct pool info before performing init_pool, open_position, close_position
- Show a prompt to ask user to verify pool information before proceeding 

## Screenshots
image

image

The pictures show a `config.json` file  with incorrect token order of a pool. Following Whirlpool requirements, the token need to be sorted in `canonical order`. 

## Reversing mechanism 
- if token order is not reversed, get the data and proceeded as normal
- if token is reversed, do the following 

```
 correctInitialAmountBPerA = 1 / initialAmountBPerA;
 correctLowerBPerAPrice = 1 / upperBPerAPrice;
 correctUpperBPerAPrice = 1 / lowerBPerAPrice;
```    
The detail of the scripts is in `scripts/create_pool/utils/pool.ts`

## Checklist
  <pre lang="checklist">
  https://checklist.remop.info/remitano-remitano/pr-nemoswap-13
  </pre>

### CODE REVIEW

File: scripts/create_pool/config.json
Comment (trathailoi) [inline scripts/create_pool/config.json:1]: mình chỉ sửa logic chứ ko sửa config chỗ này thì discard changes của file này nhen em

File: scripts/create_pool/utils/pool.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    rl.question(
+      ` ${message} \n
+        This is your pool information: 
+          -----------------------------------------------
+          token_a: ${poolInfo.tokenMintA}  
+          token_b: ${poolInfo.tokenMintB} 
+          tick_spacing: ${poolInfo.tickSpacing} 
+          price_b_per_a: ${poolInfo.initialAmountBPerA.toFixed(6)} 
+          lower_b_per_a_price: ${poolInfo.lowerBPerAPrice.toFixed(6)} 
+          upper_b_per_a_price: ${poolInfo.upperBPerAPrice.toFixed(6)}
```
Comment (trathailoi) [inline scripts/create_pool/utils/pool.ts:28]: đảo vị trí 2 line này cho nhau để nó intuitive hơn nhen em ơi

Comment (kientt6034) [reply scripts/create_pool/utils/pool.ts:28]: dạ vâng a

Comment (kientt6034) [reply scripts/create_pool/utils/pool.ts:28]: ơ mà a, là mình để upper price trước lower price ạ? Em đang để theo thứ tự lúc mình điền file config json

**Review by trathailoi — APPROVED** [review]

**Pull request merged**